style.css

```diff
@@ -1,171 +1,218 @@
+        color: limegreen;
+        font-size: 12px;
+    }
+    
+    .hh {
+        margin-left: 45px;
+    }
+    
+    .mm {
+        margin-left: 80px;
+    }
+    
+    .ss {
+        margin-left: 70px;
+    }
+    
+    .cs {
+        margin-left: 45px;
+    }
+    
+    .controls {
+        margin-left: 150px;
+        margin-top: 30px;
+    }
+    
+    button {
+        box-shadow: 5px 5px 5px #888888;
+    }
+    
+    .watches {
+        text-align: center;
+    }
+    
+    .watch1 {
+        text-align: left;
+        display: inline-block;
+        width: 50%;
+    }
+    
+    .watch2 {
+        text-align: left;
+        display: inline-block;
+        width: 40%;
+    }
+    
+    .watch3 {
+        text-align: left;
+        display: inline-block;
+        width: 40%;
+    }
+    
+    .text {
+        color: #fff;
+        font-size: 25px;
+    }
+    
+    .textbox {
+        font-size: 18px;
+    }
+    
+    #go {
+        background-color: limegreen;
+    }
+    
+    #stop {
+        background-color: red;
+        display: none;
+    }
+    
+    #clear {
+        background-color: yellow;
+        display: none;
+    }
+    
+    #plus {
+        background-color: limegreen;
+    }
+    
+    #minus {
+        background-color: red;
+    }
+    /* Buttons */
+    
+    button,
+    .input {
+        transform: scale(0.9);
+    }
+    
+    button:hover,
+    .input {
+        transform: scale(1);
+        opacity: 0.8;
+    }
+
+    /* Todo List */
+    #todo_container {
+        width: 820px;
+        height: 410px;
+        font-size: 40px;
+        background-color: black;
+        color: limegreen;
+        padding: 10px 25px;
+        border-radius: 10px;
+        margin: 50px auto;
+        font-size: 0;
+    }
+    
+    #todo_list {
+        height: 200px;
+        overflow-y: auto;
+    }
+    
+    #taskInput {
+        height: 2rem;
+        font-size: 1.2rem;
+        width: 25rem;
+        font-family:Arial, Helvetica, sans-serif;
+        margin-bottom: 0.5rem;
+    }
+    
+    .todoHeading{
+        width: 10rem;
+        font-size: 3rem;
+        font-family: 'Ubuntu', sans-serif;
+        color: white;
+        border-bottom: 2px solid limegreen;
+        border-bottom-style: ridge;
+        margin: 0% auto;
+    }
+
+    #checkButton{
+        width: 1.4rem;
+        height: 1.1rem;
+        border-radius: 0;
+        border: 0;
+        margin: 0rem 0.8rem;
+    }
+
+    #deleteButton{
+        width: 1.5rem;
+        height: 2rem;
+        font-size: 1.2rem;
+        color: red;
+        transition: ease-in-out 0.2s;
+    }
+
+    #deleteButton:hover{
+        background-color: red;
+        color: white;
+        border: none;
+    }
+
+    #prb_cntr {
+        width: 420px;
+        font-size: 40px;
+        background-color: black;
+        color: limegreen;
+        padding: 50px 20px;
+        border-radius: 10px;
+        margin: 50px auto;
+        text-align: center;
+    }
```

Comment:
Please make this prb_cntr class have the same styling which is there now in the Raghwendra-Dey/productivity_meter:master . Since this is changed a bit after #36 .